GraphQL/Models/HandednessEnumerationGraphType.cs

```diff
@@ -0,0 +1,7 @@
+﻿using GraphQL.Types;
+using Lombiq.TrainingDemo.Models;
+
+namespace Lombiq.TrainingDemo.GraphQL.Services
+{
+    public class HandednessEnumerationGraphType : EnumerationGraphType<Handedness> { }

```

Comment:
Erre a fájlra valahogy hivatkozzunk a `PersonPartObjectGraphType`-ból. Annyi is elég, hogy kb. "check out and come back".